frappe/desk/doctype/number_card/number_card.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020, Frappe Technologies and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+import frappe
+from frappe.model.document import Document
+
+class NumberCard(Document):
+	pass
+
+
+def get_permission_query_conditions(user):
+
+	if not user:
```

Comment:
`user` is a mandatory argument in the function, we can set its default to `None` otherwise the check is not needed